pdk/proto/BUILD

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_hdl//pdk/proto:build_defs.bzl", "extract_lef_and_liberty")
+load("@rules_python//python:proto.bzl", "py_proto_library")
+
+proto_library(
+    name = "pdk_info_proto",
+    srcs = ["pdk_info.proto"],
+)
+
+cc_proto_library(
+    name = "pdk_info_cc_proto",
+    deps = [":pdk_info_proto"],
+)
+
+py_proto_library(
+    name = "pdk_info_py_proto",
+    deps = [":pdk_info_proto"],
+)
+
+extract_lef_and_liberty(
```

Comment:
@QuantamHD not sure this is the best place for those?

Reply:
Why not put these in the sk130 build file?